docker/nitro/Dockerfile

```diff
@@ -51,14 +51,21 @@ RUN wget https://github.com/openssl/openssl/archive/refs/tags/openssl-3.0.1.tar.
     make -j $(nproc) && \
     make install_sw install_ssldirs && \
     cd .. && \
-    rm -rf openssl-openssl-3.0.1 openssl-3.0.1.tar.gz
+    rm openssl-openssl-3.0.1 openssl-3.0.1.tar.gz
```

Comment:
This change seems suspicious since `openssl-openssl-3.0.1` is a directory.
Side question should we update to version 3.0.6? There is at least one DoS fix in [3.0.2](https://github.com/openssl/openssl/blob/master/CHANGES.md#changes-between-301-and-302-15-mar-2022) that might be relevant.

Reply:
Good catch

Reply:
How about we bump openssl in a separate PR?

Reply:
That seems like the correct thing to do

Reply:
Done in #557